ddtrace/internal/telemetry/data/dependency.py

```diff
@@ -0,0 +1,14 @@
+from typing import TypedDict
+
+
+# Stores the name and versions of python modules
+Dependency = TypedDict("Dependency", {"name": str, "version": str})
+
+
+def create_dependency(name, version):
```

Comment:
For functionality no but it was added to make this code more readable. I could use `Dependency(name=", version="")` instead of this helper. I just removed it